l10n_th_tin_data/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Poonlap V.
+# Licensed AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Thai Localization - TIN data",
+    "version": "13.0.1.0.0",
+    "author": "Poonlap V.,Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/l10n-thailand",
+    "license": "AGPL-3",
+    "category": "Localisation/Asia",
+    "summary": "Verify Tax ID and get partner's information from web services.",
+    "depends": ["l10n_th", "l10n_th_partner", "l10n_th_base_location"],
```

Comment:
```suggestion
    "depends": ["l10n_th_partner", "l10n_th_base_location"],
```
Thailand's COA should not be specific. In real life, it can be varied. I guess is is the one causing error on Travis.

Reply:
From my testing, if 'l10n_th' is not enable,  I can not save the new contact. 
It will show 'the following fields are invalid Account Receivable'.  
I will double check again.

Reply:
If you check "demo" when create new database. And then try install module "account". You will see a simple CoA. By default, this is the CoA that CI will use. So, our test script should use it too.

So, I can confirm l10n_th shouldn't be installed.